triplea_maps.yaml

```diff
@@ -938,28 +938,29 @@
   description: |
     <br>This is a MAP SKIN for "Diplomacy"
     <br>A very basic style.  I made this one for people who hate the cursive writing of the other 3 skins.  It has a simple font, no textures, and no cursive.
-  version: 1
+  version: 1.7
```

Comment:
We have version numbers in the XML file as well. That version number seems to be confused with the one here.  We are due to clean that system up, it is confusing.

The version number in this file is used only on the download screen, nowhere else. The values should be incrementing natural numbers (eg: 1, 2, 3, 4)
The reason for natural numbers here is there is no real meaning behind point releases. The only use for this number is to indicate a new version is available for download. The XML versions are perhaps used further for compatibility notifications.